starwars/src/components/Character.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const Character = props => {
+  return (
+    <div className='char-container'>
+    <div className="character">
+      <div className="character-name"> 
+      <ul>
+      <li>{props.character.name}</li>
+      </ul>
+      </div>
+      <div className="character-data">
+       <span>Gender:</span> {props.character.gender}<br>
+       </br>
```

Comment:
Instead of `<br></br>`, you can achieve the same effect with `<br />`
```suggestion
       <br />
```